runtime/jvmti/j9jvmti.tdf

```diff
@@ -617,3 +617,7 @@ TraceEvent=Trc_JVMTI_jvmtiAddCapabilities_turnOfRomMethodSorting Overhead=1 Leve
 
 TraceEntry=Trc_JVMTI_jvmtiHookModuleSystemStarted_Entry Overhead=1 Level=1 Noenv Template="ModuleSystemStarted"
 TraceExit=Trc_JVMTI_jvmtiHookModuleSystemStarted_Exit Overhead=1 Level=1 Noenv Template="ModuleSystemStarted"
+
+TraceEntry=Trc_JVMTI_jvmtiSetHeapSamplingInterval_Entry Overhead=1 Level=5 Noenv Template="SetHeapSamplingInterval env=%p samplingInterval=%d"
+TraceEntry=Trc_JVMTI_jvmtiSetHeapSamplingInterval_Exit Overhead=1 Level=5 Noenv Template="SetHeapSamplingInterval: returning %d"
```

Comment:
Presumably should be TraceExit

Reply:
Also, below, should there not be a matching enter?

Reply:
yeah, fixed. Please review.